### PR TITLE
add script for the content content languages

### PR DESCRIPTION
**Pull request by sewagodimo**

### CODE REVIEW

File: gem/management/commands/change_content_language.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                        "Invalid language primary keys were entered" +
+                        str(old_lang_pk) + " and " + str(new_lang_pk)))
+                else:
+                    pages = models.MoloPage.objects.all()
+                    for page in pages:
+                        if hasattr(page.specific, 'language'):
+                            if page.specific.language == old_lang:
+                                page.specific.language = new_lang
+                                page.specific.save()
+                                page.save()
```
Comment (codiebeulaine) [inline gem/management/commands/change_content_language.py]: you might need to do save_revision().publish() if the article or page is live and save_revision() if it is draft?

Comment (sewagodimo) [reply gem/management/commands/change_content_language.py]: thank you

Comment (sewagodimo) [reply gem/management/commands/change_content_language.py]: @codie I tested it and the language change doesn't require me to publish the pages again, it actually makes the changes fail for some of the content

**Comment by codiebeulaine** [comment]
👍

**Pull request merged**